releasenotes/notes/0.46/prepare-0.46.0-c3857463a3c052f7.yaml

```
@@ -0,0 +1,620 @@
+    which documents the release schedule in more detail.
+
+    .. note::
+
+      If your project depends on Qiskit, it may rely on functionality that will no longer
+      be supported in Qiskit 1.0.0. For this reason, we recommend that you proactively cap your
+      supported version to ``<1.0``. Qiskit 1.0.0 is scheduled to release approximately two weeks
+      after Qiskit 0.46.0, on 2024-02-15, and might not yet be available when
+      you read this message.
+
+    The packaging structure of Qiskit is changing in Qiskit 1.0, and unfortunately the changed
+    requirements cannot be fully communicated to ``pip``, especially if ``pip install --upgrade``
+    commands are run after the environment has been initially configured.  All versions of Qiskit
+    prior to 1.0 (including this one) have an installation conflict with Qiskit 1.0 that ``pip``
+    will not resolve.
+
+    If ``import qiskit`` raises an :exc:`ImportError` for you, your environment is in an invalid
+    state, and versions of Qiskit 0.45/0.46 and 1.0 are both reachable, which will result in subtley
+    broken code.  You will need to create a new virtual environment, and ensure that *only* one of
+    the two versions are installed.  In particular, if you are intending to install Qiskit 1.0,
+    you must have no packages that depend on ``qiskit-terra`` installed; these packages are
+    incompatible with Qiskit 1.0 and must be updated.  If you are intending to install Qiskit 0.45
+    or 0.46, you must ensure that you have nothing attempting to install ``qiskit>=1.0``.
+
+    If you develop a library based on Qiskit and you still have a dependency on ``qiskit-terra``,
+    you should urgently release a new package that depends only on ``qiskit``.  Since version 0.44,
+    the ``qiskit`` package contained only the ``qiskit-terra`` compiler core (the component that is
+    now simply called "Qiskit"), so if your minimum version is ``0.44``, you can safely switch a
+    ``qiskit-terra>=0.44`` dependency to ``qiskit>=0.44`` with no change in what will be installed.
+    For more detail and recommendations for testing and preparation, see the
+    `section for developers of the migration guide <https://qisk.it/1-0-packaging-migration#for-developers>`__.
+
+features:
+  - |
+    Added a new function :func:`~.qs_decomposition` to :mod:`qiskit.synthesis` which
+    will apply Quantum Shannon Decomposition of arbitrary unitaries.
+
+  - |
+    A new :mod:`qiskit.providers.basic_provider` module has been introduced
+    to replace ``qiskit.providers.basicaer``. This module contains provider
+    tools that mirror those of the ``BasicAer`` provider and offers a single,
+    non-efficient, statevector based simulator: :class:`.BasicSimulator`.
+    This simulator based on the :class:`.BackendV2` interface and is
+    exclusively intended for testing and simple prototyping, for more
+    advanced simulation capabilities, please refer to the ``qiskit-aer``
+    package.
```

Maybe we should add a link here to the deprecation note?

```suggestion
    package. See the ``BasicAer`` deprecation note for migration guidelines.
```